apps/lib/apps.c

```diff
@@ -1003,10 +1003,18 @@ int load_key_certs_crls(const char *uri, int format, int maybe_stdin,
         ctx = OSSL_STORE_open_ex(uri, libctx, propq, get_ui_method(), &uidata,
                                  params, NULL, NULL);
     }
-    if (ctx == NULL)
+    if (ctx == NULL) {
+        if (!quiet)
+            BIO_printf(bio_err, "Could not open file or uri for loading");
         goto end;
-    if (expect > 0 && !OSSL_STORE_expect(ctx, expect))
+    }
+    if (expect > 0 && !OSSL_STORE_expect(ctx, expect)) {
+        if (!quiet)
+            BIO_printf(bio_err, "Could not find");
+        if (failed == NULL)
+            failed = "anything";
```

Comment:
These two new lines are not needed - 
by this point in the control flow, `failed` cannot be NULL due to above lines 945 and following:
```
    if (failed == NULL) {
        if (!quiet)
            BIO_printf(bio_err, "Internal error: nothing to load from %s\n",
                       uri != NULL ? uri : "<stdin>");
        return 0;
    }
```

Reply:
Ah, right.